src/Tizen.NUI.Components/Controls/Switch.cs

```diff
@@ -62,12 +62,20 @@ public Switch(SwitchStyle switchStyle) : base(switchStyle)
             Initialize();
         }
 
+        /// <summary>
+        /// Initilize AT-SPI object.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public override void OnInitialize()
         {
             base.OnInitialize();
             SetAccessibilityConstructor(Role.ToggleButton);
         }
 
+        /// <summary>
+        /// Prevents from showing child widgets in AT-SPI tree.
```

Comment:
This summary is probably for the AccessibilityShouldReportZeroChildren

Reply:
Done.